paxml/setup_jax.py

```diff
@@ -15,20 +15,32 @@
 
 r"""Utilities to set up JAX global configs."""
 
+import dataclasses
 from typing import Optional
 
 from absl import logging
 import jax
 from praxis import py_utils
 import tensorflow.compat.v2 as tf
 
+@dataclasses.dataclass
+class JaxDistributedOptions:
+  coordinator_address: int
```

Comment:
address should be a string


def initialize(coordinator_address: Optional[str] = None,
               num_processes: Optional[int] = None,
               process_id: Optional[int] = None,
               local_device_ids: Optional[Union[int, Sequence[int]]] = None):